go/stats/counters.go

```diff
@@ -224,3 +224,13 @@ func NewMultiCountersFunc(name string, labels []string, f CountersFunc) *MultiCo
 	}
 	return t
 }
+
+var escaper = strings.NewReplacer(".", "\\.", "\\", "\\\\")
```

Comment:
I created a benchmark for MultiCounters:

```go
var benchMultiCounter = NewMultiCounters("benchMulti", []string{"call", "keyspace", "dbtype")

func BenchmarkMultiCounters(b *testing.B) {
	clear()
        key := []string{"execute-key-ranges", "keyspacename", "replica"}
	benchCounter.Add(key, 1)
	b.ResetTimer()

	b.RunParallel(func(pb *testing.PB) {
		for pb.Next() {
			benchCounter.Add(key, 1)
		}
	})
}
```

before:
`BenchmarkMultiCounters-3        300000000              186 ns/op`

after:
`BenchmarkMultiCounters-3        200000000              261 ns/op`

So this adds only 75 ns to a 186 ns operation (+40%) in the normal case where there are no dots or backslashes in the key. Even if it ran 100 times per query it would only add 7.5 microseconds. Note that the whole key is already being copied by strings.Join; this is only adding a pass to scan the key for 2 characters.